cogs/gamerwords.py

```diff
@@ -267,8 +294,21 @@ async def handle_new_gamer_message(self, message):
 				allowed_mentions=allowed_mentions
 			)
 
+	async def dl_attach(self, attach, message):
```

Comment:
this function should be defined inside  handle_new_gamer_message as it's called nowhere else and raises an exception class that is only caught there